src/Deferred.php

```diff
@@ -14,6 +14,9 @@ public function __construct(callable $canceller = null)
         $this->canceller = $canceller;
     }
 
+    /**
+     * @inheritdoc
+     */
```

Comment:
Yes, standalone inheritdoc is pretty useless and as the [FIG proposal states](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#62-using-the-inheritdoc-inline-tag-to-augment-a-description), it should only be used to augment the description of the parent element.